Dockerfile

```diff
@@ -39,6 +40,10 @@ COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
 COPY --chown=nextjs:nodejs next-logger.config.js /app/
 COPY --chown=nextjs:nodejs pino-pretty-transport.js /app/
 
+# Create media directory
+RUN mkdir -p $MEDIA_PATH
+RUN chown -R nextjs:nodejs $MEDIA_PATH
+
```

Comment:
This is fine but when actually running the docker image in production a volume will probably be mounted in this location which "overwrites" the folder created here. That is fine but that means that the chown will not apply and the folder will probably be owned by root as default which needs to be handled when the application starts somehow.

Reply:
Hmm, that might be difficult as the container doesn't run as the root user. Will have to do some research on this.

Reply:
After testing it seams to work